SYCL/ESIMD/histogram_256_slm.cpp

```diff
@@ -174,14 +178,21 @@ int main() {
       etime = esimd_test::report_time("kernel time", e, e);
       if (iter > 0)
         kernel_times += etime;
+      else
+        start = timer.Elapsed();
     }
   } catch (cl::sycl::exception const &e) {
     std::cout << "SYCL exception caught: " << e.what() << '\n';
     return e.get_cl_code();
```

Comment:
Could you please remove such calls as this one. I tis obsolete CL API, which causes warnings
```suggestion
    return 1;
```

Reply:
BTW, I forgot about this comment when approved 7f6ab8de10f0596665011f41af34c2c5e78e2d94
Would you please also remove get_cl_code() call as well?

Reply:
Should I change it to e.code().value() instead?

Reply:
I have changed it as you suggested.